src/zope/publisher/tests/test_browserrequest.py

```diff
@@ -220,6 +227,38 @@ def testFileUploadPost(self):
         # Test that we can actually read the file data
         self.assertEqual(request.form['upload'].read(), b'Some data')
 
+    def testFileUploadPost(self):
+        """Produce a Fieldstorage with a file handle that exposes
+        its filename."""
```

Comment:
Remove the docstring?  It's not accurate, and some test runners like to use `__doc__` instead of `__name__`, obscuring which test it is running.  This is especially bad when you have multiple tests with the same docstring, which is the case here.